src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashRepository.java

```diff
@@ -311,7 +322,8 @@ public void addFutureBuildTasks(Collection<StashPullRequestResponseValue> pullRe
     }
   }
 
-  public void deletePullRequestComment(String pullRequestId, String commentId) {
+  public void deletePullRequestComment(String pullRequestId, String commentId)
+      throws StashApiException {
```

Comment:
do we need to add this to the signature?

Reply:
Yes. The callers are in a better position to handle the exception. In particular, `StashBuildListener` can post the issue to the build output.